packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -29,8 +29,8 @@ void main() {
     expect(tester.takeException(), isAssertionError);
   });
 
-  // Regression test for https://github.com/flutter/flutter/issues/123064
   testWidgets('shadow effect is not doubled', (WidgetTester tester) async {
+  // Regression test for https://github.com/flutter/flutter/issues/123064
```

Comment:
nit: indentation (here and below)
```suggestion
  testWidgets('shadow effect is not doubled', (WidgetTester tester) async {
    // Regression test for https://github.com/flutter/flutter/issues/123064
```